test/src/org/exist/http/RESTServiceTest.java

```diff
@@ -33,6 +33,7 @@
 import java.net.URLEncoder;
 import javax.xml.parsers.ParserConfigurationException;
 
+import org.eclipse.jetty.http.HttpStatus;
```

Comment:
better to use https://docs.oracle.com/javaee/6/api/javax/servlet/http/HttpServletResponse.html#SC_NOT_FOUND ?

Reply:
I don't see our dependency on Jetty going away, but I am happy to revise it. Do you wish me to do so?